### ISSUE TITLE
VarIdx's should be controlled by a flag

### ISSUE THREAD

**Issue by behdad** [BlackFoundryCom/variable-components-spec#9]
Not every font using smart components will be variable... I think a flag should control whether VarIdx values are present.

This also means the font should work without fvar.

**Comment by justvanrossum** [comment]
> Not every font using smart components will be variable... 

True. (I don't like to call them "smart". They're very variable, but not smart by themselves.)

> I think a flag should control whether VarIdx values are present.

I don't think this is true when nesting variable components, but I need to wrap my head around it, so I'm not confident.
